openfasoc/generators/glayout/glayout/llm/rag_data/CommonDrain.md

```diff
@@ -0,0 +1,4 @@
+# Common Drain Amplifier
+The purpose of a common drain amplifier, aka a source follower, is to provide buffering. It has high input impedance, low output impedance, and a voltage gain close to unity.
```

Comment:
maybe provide a separate example of a source follower being used in buffering. The LLM might place just a common drain stage as a buffer, which isn't the most ideal